src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestFixture.scala

```diff
@@ -25,6 +30,40 @@ trait AgoraTestFixture {
   val agoraBusiness: AgoraBusiness = new AgoraBusiness(permsDataSource)
   val permissionBusiness: PermissionBusiness = new PermissionBusiness(permsDataSource)
 
+  var waasMockServer: ClientAndServer = _
+  val waasRequest = request()
+    .withMethod("POST")
+    .withPath("/api/womtool/v1/describe")
+
+  val mockAccessToken = AgoraConfig.mockAccessToken
+
+  def startMockWaas() = {
+    waasMockServer = startClientAndServer(waasMockServerPort)
+    addSubstringResponse(payload1.get, payload1DescribeResponse)
+    addSubstringResponse(payload2.get, payload2DescribeResponse)
+    addSubstringResponse(payloadWithValidPersonalDockerImageInWdl.get, genericDockerPayloadDescribeResponse)
+    addSubstringResponse(payloadWithValidOfficialDockerImageInWdl.get, genericDockerPayloadDescribeResponse)
+    addSubstringResponse(payloadWdlBadVersion.get, badVersionDescribeResponse)
+    addSubstringResponse(badPayload.get, malformedPayloadDescribeResponse)
+    addSubstringResponse(payloadWdl10.get, goodWdlVersionDescribeResponse)
+  }
+
+  def addSubstringResponse(payload : String, response : String) = {
+    // necessary to declare this here in order to disambiguate overloaded method
+    val p2 : BodyWithContentType[String] = new StringBody(payload, true)
+    waasMockServer.when(
+      waasRequest
+        .withBody(p2)
+        )
+      .respond(
+        org.mockserver.model.HttpResponse.response().withBody(response).withStatusCode(OK.intValue).withHeader("Content-Type", "application/json"))
+
+  }
```

Comment:
nit: this isn't actually a substring check, is it? It checks for equality on the body.

Reply:
it's actually a substring.  the body of the post is actually a multi-part form response and so the body contains all the content separators, etc in addition to the payload.